designsafe/static/scripts/data-depot/components/projects/curation-directory/curation-directory.component.html

```diff
@@ -4,8 +4,8 @@
         <div class="prj-head-container">
             <span class="prj-head-title">
                 {{$ctrl.browser.project.value.projectId}} | <strong>{{ $ctrl.browser.project.value.title }}</strong>
-                <a class="prj-head-edit" ng-click="$ctrl.manageProject($event)"><strong>Edit Project</strong></a>
             </span>
+            <a class="prj-head-edit" ng-click="$ctrl.manageProject($event)"><strong>Edit Project</strong></a>
```

Comment:
I'm kinda confused as to why I see this change only for the `curation-directory` template and not the `project-view` template. I'm assuming that the Edit Project button would be in two different places before, but apparently this change is already made in the `project-view`?

Reply:
I think the only other places that need this change are in the following templates:
`pipeline-amend.template.html` - this is in the "Amend" pipeline for other projects that have been published. I can show you how to get to this page if you don't have an "other" project to use
`pipeline-project.template.html` - This is in the normal publication pipeline after the selection step.

Reply:
the change for the `project-view` template was already merged in, it's in staging currently if you want to check.
I'll check those other templates and look at the other removals you mentioned.

Reply:
Do we need to not allow a user to do an "edit project metadata" from the `pipeline-amend.template.hmtl`?  When I did these updates, I did a search for "edit project" and I saw this one, but I think it's more of amending the information for the project metadata, which I think should be allowed?

It does look like I somehow missed `pipeline-project.template.html`, so updating that file.